features/admin/orphan_lists.feature

```diff
@@ -158,7 +158,7 @@ Feature:
     Then I click the "Upload" button
     Then I should see "محمد"
     And I should see "John Doe"
-    And I should see "January 02, 2012"
+    And I should see "2012-01-02"
```

Comment:
Is this related to https://github.com/AgileVentures/osra/pull/260?  I remember the client said they want day, month, year everywhere so I made that global, even across the AA side.  But, I didn't see any tests fail